Automattic/wp-calypso
# Stats: Update Interval dropdown to preserve date ranges

Related to #83150.

## Proposed Changes

Replaced the Interval component with a custom list so we can control the routing directly. With the new component in place, selections will preserve the current date range.

**Note:** Based on work done in #83171 so that should be merged first.

## Testing Instructions



1. Open Live branch to the **Stats → Traffic** page.
2. Enable date picker: `?flags=stats/date-control`
3. Select a shortcut from the date picker to update the URL params.
4. Test and confirm the interval dropdown works and maintains the date range.

## Pre-merge Checklist
